db/schema.rb

```diff
@@ -208,6 +208,16 @@
     t.index ["uuid"], name: "index_document_capture_sessions_on_uuid"
   end
 
+  create_table "duplicate_profile_confirmations", force: :cascade do |t|
```

Comment:
It seems like there are a few changes to this file that probably shouldn't be here.